tests/foreman/sys/test_rename.py

```diff
@@ -38,7 +38,7 @@ class TestRenameHost:
     """Implements ``katello-change-hostname`` tests"""
 
     @pytest.mark.skip_if_open("BZ:1925616")
-    def test_positive_rename_satellite(self, module_org, module_product):
+    def test_positive_rename_satellite(self, module_org, module_product, destructive_sat):
```

Comment:
Is this test been auto marked as destructive ?

Reply:
there is no auto-mark for destructive

Reply:
@JacobCallahan I ll implement the automark wherever `destructuve_sat` fixture is being called, the tests will be automatically marked for `destructive`.

BTW, just for the purpose of this test, we should mark it with `destructive` manually.